### DISCUSSION TITLE
First official release - Release Candidate

### DISCUSSION THREAD

**Discussion by ivomp** [OpenManufacturingPlatform/iotcon-opc-ua-connector-dotnet#52] [Announcements]
*Please use discussion

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenManufacturingPlatform/iotcon-opc-ua-connector-dotnet/releases/tag/0.1.0-RC'>First official release - Release Candidate</a>.</em>

**Comment by ctron** [comment]
What is the outcome of the license discussion? Unless I misread something, I believe that the OPC UA library `OPCFoundation.NetStandard.Opc.Ua` used by the connector is licensed under the GPL 2.0: https://github.com/OPCFoundation/UA-.NETStandard/blob/8c1a659ecf5c1616b3b7f132706324b90f9ff015/LICENSE.txt#L38-L41

So is the GPL 2.0 now an approved license (which I would appreciate)? What about the other licenses we discussed in the base?

**Reply by ivomp** [reply]
From what I understood, we can release the source code as is. But we may not release a runnable component (due to the licenses). 
So the public is free to download it, compile and run it by them self. Then we are still not in breach of any licensing.

But maybe @sbuckel  can gives us more clarity on this topic?

**Reply by sbuckel** [reply]
Please see the OMP License policies: https://github.com/OpenManufacturingPlatform/omp-the-way-we-work-private/blob/development/Rules/rules_of_engagement.md#omp-software-license-policy (internal)

Point 2: "It is not recommended to pull software code, under different license than the Project License, into the project repository. Use linked libraries instead."
Point 4: "It is a good practice to point users to the libraries so they can compile them on their own"

We release only the software code, no compiled sofware and binaries.

**Comment by jpradocueva** [comment]
Unfortunately, Linux Foundation cannot provide license advice. 

In any case, by reading the information provided in this discussion,  I believe that this topic needs to be flagged to the Organization team as per [OMP Software License Policy](https://github.com/OpenManufacturingPlatform/omp-the-way-we-work-private/blob/publication/Rules/rules_of_engagement.md#omp-software-license-policy).
The connector is using a library that is released under GPL 2.0. The Organization Team needs to be aware of the implications.

Also, don't you think that the project should create the NOTICE file? As suggested in the second bullet  (`2. Linked Libraries & 3rd Party Software`) of the [OMP Software License Policy](https://github.com/OpenManufacturingPlatform/omp-the-way-we-work-private/blob/publication/Rules/rules_of_engagement.md#omp-software-license-policy),
